### PR TITLE
WL-3764 Added image and style for the 'oxford-talks' quick links.

### PR DESCRIPTION
**Pull request by ouit0408**
Wl-3632 used individual service logo "favicons" in Quick Links menu ,where available,
and default for the rest. Fix is linked to the pull request https://github.com/ox-it/wl-portal/pull/51

### CODE REVIEW

**Comment by buckett** [comment]
The custom icons are probably better going outside the silk icon folder as that is just a copy of the famfamfam silk icons and it's generally better if we can keep our changes separate from upstream projects.

File: library/src/webapp/skin/neo-ox/portal.css [hunk 1]
```diff
@@ -50,6 +50,27 @@ body.portalBody {
...
+
+.researcher-training{
+    background-image: url(/library/editor/ckextraplugins/researcher-training-tool/icons/researcher-training-tool.png) !important;
+}
+
+.solo{
+	background-image: url(/library/editor/ckextraplugins/solo-citation/icons/solo-citation.png) !important;
+}
+
+.oxam{
```
Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: It's better to try to have slightly more specific CSS classes, so it's clearer where they should be used, but also less likely to clash with other elements.

**Comment by buckett** [comment]
`library/src/webapp/skin/neo-ox/images` is a much better place to put the icons.

**Comment by ouit0408** [comment]
placed all quick links icons inside the library/src/webapp/skin/neo-ox/images' and have pushed the code.

**Comment by buckett** [comment]
The original icons are still in the pull request.

**Comment by ouit0408** [comment]
deleted icons from the folder silk and pushed my changes.

**Pull request merged**